platform/ios/scripts/package.sh

```diff
@@ -207,17 +222,18 @@ SEM_VERSION=$( git describe --tags --match=ios-v*.*.* --abbrev=0 | sed 's/^ios-v
 SHORT_VERSION=${SEM_VERSION%-*}
 if [[ ${BUNDLE_RESOURCES} ]]; then
     cp -pv LICENSE.md "${OUTPUT}/static/${NAME}.framework"
-    cp -rv platform/ios/app/Settings.bundle "${OUTPUT}/static/${NAME}.framework"
+    cp -rv platform/ios/app/Settings.bundle ${STATIC_SETTINGS_DIRECTORY}
```

Comment:
Ok, I think I'm finally with you @1ec5. So we would like something like (focusing on the static side of the fence and the files moving around in this change):

- ifabric
  - pkg/static/Mapbox.framework (holds **everything**)

- ipackage* flavors
  - pkg/
    - Mapbox.bundle
    - Settings.bundle
    - static/
  - pkg/static/
    - Mapbox.framework

I'll adjust accordingly.